src/v2/Apps/Settings/Routes/DeleteAccount/DeleteAccountRoute.tsx

```diff
@@ -114,3 +154,14 @@ export const DeleteAccountRoute: FC = () => {
     </GridColumns>
   )
 }
+
+export const DeleteAccountRouteFragmentContainer = createFragmentContainer(
+  DeleteAccountRoute,
+  {
+    me: graphql`
+      fragment DeleteAccountRoute_me on Me {
+        hasPassword
+      }
+    `,
+  }
+)
```

Comment:
want to point out a strange thing. when @ovasdi and i were pairing on this, we weren't able to see this graphql query in Chrome devtools, in `console` or `network` tab. yet somehow it works.

Reply:
If visiting this page directly, the reasoning is this query executes during server-side render, and during that phase we have relay logging turned off. After the initial (1st) page render we switch to client-side rendering for all subsequent (modern) pages and should be able to inspect all network requests via the relay logger in the browser console. 

If wanting to debug during the SSR pass, y'all can play with [this line](https://github.com/artsy/force/blob/4581f73773c01d4fcc780197ed8d2993d47745b5/src/v2/System/Relay/createRelaySSREnvironment.ts#L118) where we set logging up.

Reply:
ah, that explains it. if i log out/in and click through from homepage to account-delete page, i see the query consistently:
<img width="1174" alt="Screen Shot 2022-05-20 at 3 54 46 PM" src="https://user-images.githubusercontent.com/63004951/169601932-d81ed1fa-7c33-47ba-bdc4-793dd5e84f92.png">

Reply:
thanks!